test/plugins/individual-collateral/morpho-aave/constants.ts

```diff
@@ -0,0 +1,17 @@
+import { bn, fp } from '../../../../common/numbers'
+import { networkConfig } from '../../../../common/configuration'
+
+// Mainnet Addresses
+export const USDC_USD_PRICE_FEED = networkConfig['1'].chainlinkFeeds.USDC as string
+export const ETH_USD_PRICE_FEED = networkConfig['1'].chainlinkFeeds.ETH as string
+export const USDC = networkConfig['1'].tokens.USDC as string
+
+export const PRICE_TIMEOUT = bn(604800) // 1 week
+export const ORACLE_TIMEOUT = bn(86400) // 24 hours in seconds
+export const ORACLE_ERROR = fp('0.005')
+export const DEFAULT_THRESHOLD = bn(5).mul(bn(10).pow(16)) // 0.05
+export const DELAY_UNTIL_DEFAULT = bn(86400)
+
+export const USDC_DECIMALS = bn(6)
+
+export const FORK_BLOCK = 17528677
```

Comment:
there's another fork block number in `fork-block-numbers.ts`, not sure which is being used but there should probably only be 1